Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -1053,11 +1072,17 @@ chooseYourRideConfig state = ChooseYourRide.config
   {
     rideDistance = state.data.rideDistance,
     rideDuration = state.data.rideDuration,
-    quoteList = state.data.specialZoneQuoteList,
+    -- quoteList = state.data.specialZoneQuoteList,
+    quoteList = [ChooseVehicle.dummyBusConfig],
```

Comment:
why we are passing dummy and not what we get from api response, if variable name does not satisfy the condition just give generic name, it will sure break for other conditions, please check

Reply:
Right now we don't have the BPP for this, so we won't get any response if we try to use actual APIs, that's why using dummy data for the time being. Will update it once we have integrated `BAP<->BPP`